tests/k8s/conftest.py

```diff
@@ -19,6 +19,11 @@ def logger():
     return logger
 
 
+@pytest.fixture()
```

Comment:
fixture not necessary

Reply:
I actually think it a safe way to get the current directory relativ to the test folder in all the test of this module without duplicating code. The alternative would be to put `cur_dir = os.path.dirname(__file__)` into all files and potentially do some `os.path.join(cur_dir, "..", xyz)` stuff. 
Can we keep it? :)